backend/README.md

```diff
@@ -28,3 +28,13 @@ POST
 registers and logins user
 returns:
 `{ "key": "some-string" }`
+
+### /api/users/pages/
+
+GET
```

Comment:
So would this just return a collection of articles with the same title then? Not entirely clear.